src/main/webapp/projectReview/ProjectReview.js

```diff
@@ -209,37 +209,48 @@ class ProjectReview extends Component {
     return value === '' || value === null || value === undefined;
   }
 
-  approveRevision = (e) => () => {
+  approveRevision = () => {
     this.setState({ disableApproveButton: true })
-    const data = { projectReviewApproved: true }
+    const data = { projectReviewApproved: 'true' }
```

Comment:
In general, I prefer booleans to really be booleans. I'm OK with this change for this PR, but in the future, we should try to keep the types as clean as possible.

Reply:
We agree on that. This value comes from key / value pair in issue_extra_properties where both key and value are stored as string, since could hold any data type serialized as string. We retrieve all  extra properties at once for an issue.